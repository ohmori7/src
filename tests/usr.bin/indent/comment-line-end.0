/* $NetBSD: comment-line-end.0,v 1.3 2021/03/07 08:57:38 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Demonstrates handling of line-end comments.
 *
 * Even though this type of comments was added in C99, indent doesn't support
 * them, as of 2021, and instead messes up the code in unpredictable ways.
 */

int dummy // comment
= // eq
1 // one
+ // plus
2; // two

/////separator/////

void function(void){}

// Note: removing one of these line-end comments affects the formatting
// of the main function below.

int
main(void)
{
}
