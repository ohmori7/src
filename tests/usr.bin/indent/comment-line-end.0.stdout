/* $NetBSD: comment-line-end.0.stdout,v 1.3 2021/03/07 08:57:38 rillig Exp $ */
/* $FreeBSD$ */

/*
 * Demonstrates handling of line-end comments.
 *
 * Even though this type of comments was added in C99, indent doesn't support
 * them, as of 2021, and instead messes up the code in unpredictable ways.
 */

int		dummy //comment
= //eq
1 // one
+ //plus
2;
//two

///// separator /////

void
function(void)
{
}

/* $ FIXME: The space between 'Note: removing' must be preserved. */
/* $ FIXME: The spacing around the '-' in 'line-end' must be preserved. */
//Note:removing one of these line - end comments affects the formatting
// of the main function below.

int
/* $ FIXME: The '{' must be in column 1, not directly after the ')'. */
main(void){
}
